test/xpu/test_decomp_xpu.py

```diff
@@ -2,7 +2,7 @@
 
 import torch
 from torch.testing._internal.common_device_type import instantiate_device_type_tests
-from torch.testing._internal.common_utils import run_tests
+from torch.testing._internal.common_utils import run_tests, skipIfCrossRef
```

Comment:
Could you comment the usage of the decorator in PR description?

Reply:
ok